grakn-engine/src/main/java/ai/grakn/engine/SystemKeyspace.java

```diff
@@ -98,6 +98,9 @@ public boolean ensureKeyspaceInitialised(String keyspace) {
 
         try (GraknGraph graph = factory.getGraph(SYSTEM_GRAPH_NAME, GraknTxType.WRITE)) {
             ResourceType<String> keyspaceName = graph.getOntologyConcept(KEYSPACE_RESOURCE);
+            if (keyspaceName == null) {
+                throw new RuntimeException("Graph for keyspace [" + keyspace + "] not properly initialized. Missing keyspace name resource");
```

Comment:
It concerns be that this can happen. Why is that? 

Also, tiny poke for the sake of being consistent should this not be a `GraknException` ?